src/App.jsx

```diff
@@ -14,70 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function isSelected(good) {
+    return selectedGood === good;
+  }
```

Comment:
I would better create a const for this check inside map

`const isGoodSelected = selectedGood === good;`